prow/cmd/deck/main.go

```diff
@@ -232,6 +233,7 @@ var simplifier = simplifypath.NewSimplifier(l("", // shadow element mimicing the
 	l("github-login",
 		l("redirect")),
 	l("github-link"),
+	l("generic-provider-link"),
```

Comment:
Naming is hard.

"provider" is a very overloaded term for me, is this how you commonly refer to "github or gerrit" within prow?

Wonder if we could be more specific like `/git-provider-link`, `/git-review-link`, `/git-ui-link`, `/scm-link`

Reply:
agreed and picked `git-provider-link`